modules/demo-only/jupyter-hub/deployspec.yaml

```diff
@@ -20,12 +21,12 @@ deploy:
       - docker push $REPOSITORY_URI:latest && docker push $REPOSITORY_URI:$IMAGE_TAG
       - cd ..
       - cdk deploy --require-approval never --progress events --app "python app.py" --outputs-file ./cdk-exports.json
-      - export ADDF_MODULE_METADATA=$(python -c "import json; file=open('cdk-exports.json'); print(json.load(file)['addf-${ADDF_DEPLOYMENT_NAME}-${ADDF_MODULE_NAME}']['metadata'])")
+      - export SEEDFARMER_MODULE_METADATA=$(python -c "import json; file=open('cdk-exports.json'); print(json.load(file)['addf-${SEEDFARMER_DEPLOYMENT_NAME}-${SEEDFARMER_MODULE_NAME}']['metadata'])")
```

Comment:
Sorry for the late review. 
`addf` needs to be replaced with `SEEDFARMER_PROJECT` name here 

(OR)

you could run `seedfarmer metadata convert -f cdk-exports.json || true` command to grab the metadata and avoid running this one liner python command

Reply:
Will resolve in a new PR, thanks!